0000-Hex-Coverage-Limit.md

```diff
@@ -0,0 +1,104 @@
+# HIP XX: <--insert title
+
+- Author: @HeatedLime
+- Start Date: 5/12/2023
+- Category: Technical & Economic
+- Original HIP PR: #652
+- Tracking Issue: 
+- Voting Requirements: veIOT
+
+## Summary:
+This Helium Improvement Proposal (HIP) suggests adding a hex multiplier score to the MOBILE Proof of Coverage (PoC) modeled coverage points based on whether other coverage from Helium 5g deployments exist within that res12 hex. 
+
+## Motivation:
+HIP 74 was passed to incorporate obstruction data and radio signal power into the PoC reward model; however, it weighed all coverage within each res12 hex equally, even if multiple radios were already providing coverage within that res12 hex. This means deployers could point 5 outdoor radios in the same direction, and still be awarded full modeled coverage points for each res12 hex. 
+
+This proposal aims to improve the value of the network coverage by incentivizing users to deploy radios that minimize overlapping coverage, and encourage deployments in new areas. 
+
+## Stakeholders:
+The stakeholders of this proposal are radio deployers. 
+ 
+## Detailed Explanation:
+Currently, any redundant network coverage is still rewarded with the same. This discourages the buildout of coverage to new areas. To prevent overcrowding and overlapping of coverage in hexes, this HIP proposes to limit the amount of modeled coverage points radios are awarded to redundant coverage in res12 hexes. 
```

Comment:
Is this sentence correct? The part that I get lost in is the last bit: "radios are awarded **to** redundant coverage".
That 'to' article in particular seems like it needs to be replaced with something.

Reply:
Agree, I changed it to make it more clear.